src/login/pam_systemd.c

```diff
@@ -948,7 +948,7 @@ _public_ PAM_EXTERN int pam_sm_open_session(
 
         /* Talk to logind over the message bus */
 
-        r = pam_acquire_bus_connection(handle, &bus);
+        r = pam_acquire_bus_connection(handle, "systemd-system-bus", &bus);
```

Comment:
maybe give this systematic names:

`system-bus-pam-systemd` and `system-bus-pam-systemd-homed`